ci/android-install-sdk.sh

```diff
@@ -49,8 +49,7 @@ esac;
 
 # Try to fix warning about missing file.
 # See https://askubuntu.com/a/1078784
-mkdir ./.android/
-touch ./.android/repositories.cfg
+mkdir -p ./.android/
```

Comment:
It will create `/.android` instead of `/root/.android`, docker containers pwd is usually set to `/`.